parts/common/components.json

```diff
@@ -19,6 +19,11 @@
           "renovateTag": "registry=https://mcr.microsoft.com, name=windows/servercore",
           "latestVersion": "ltsc2022",
           "windowsSkuMatch": "23H2*"
+        },
+        {
+          "renovateTag": "registry=https://mcr.microsoft.com, name=windows/servercore",
+          "latestVersion": "ltsc2022",
```

Comment:
Change ltsc2022 to [ltsc2025](https://hub.docker.com/r/microsoft/windows-nanoserver)?

Reply:
It may need to add both ltsc2022 or ltsc2025 if WS2025 host supports both 2022 containers and 2025 containers instead of updating this line.